sklearn/datasets/species_distributions.py

```diff
@@ -81,6 +77,10 @@ def _load_coverage(F, header_length=6, dtype=np.int16):
     return M
 
 
+def _make_tuple(line):
```

Comment:
It's a bit hard to understand what this is about out of its context. Why not leave it in `_load_coverage`?